src/components/bridge/BridgeInfo.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useModal } from '../../hooks/useModal'
+import { Modal } from './modal'
+import bridgeApy from '../../assets/svg/questionmarkBridge.svg'
+
+const Container = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+  background: ${({ color }) => color};
+  border-radius: 12px;
+  padding: 16px;
+  padding-bottom: 0px;
+  font-size: 16px;
+  line-height: 21px;
+  font-weight: 500;
+  justify-content: center;
+`
+
+const Wrapper = styled('div')`
+  display: flex;
+  width: 100%;
+  text-align: center;
+  justify-content: center;
+  margin: 3px;
+  position: relative;
+  > span {
+    font-size: 14px;
+    font-weight: 400;
+    color: #9fa3c9;
+    padding-left: 10px;
+    cursor: pointer;
+  }
+`
+
+const Icon = styled('div')`
+  border-radius: 999px;
+  background-color: #ffffff17;
+  opacity: 1;
+  height: 24px;
+  width: 24px;
+  cursor: pointer;
+  position: relative;
+  > img {
+    opacity: 1;
+    top: 20%;
+    right: 22%;
+    position: absolute;
+  }
+`
+// eslint-disable-next-line react/display-name
+export default (props: any) => {
```

Comment:
we need to provide a name for the component, and props are not used so remove them and create the function with the old syntax `function name() {}` that's the style used in the codebase

Reply:
I think also function name starts with Capital letter